versioned_docs/version-8.5/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -193,23 +193,36 @@ To use these routes for the Zeebe Gateway, configure this through Ingress as wel
 
 As the Zeebe Gateway also uses `gRPC` (which relies on `HTTP/2`), [HTTP/2 Ingress Connectivity has to be enabled](https://docs.openshift.com/container-platform/latest/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress).
 
+Also, Zeebe Gateway needs to be encrypted with tls because the connection from HAProxy to the Zeebe Gateway service can use HTTP/2 only for re-encrypt or passthrough routes and not for edge-terminated or insecure routes.
```

Comment:
`Zeebe Gateway needs to be encrypted` => `Zeebe Gateway should be configured to use an encrypted connection` (to avoid confusion whether it's connection or storage encryption).

Reply:
✅